PlexAPI/Library.cs

```diff
@@ -187,7 +187,7 @@ public interface ILibrary
         /// 
         /// </remarks>
         /// </summary>
-        Task<SearchLibraryResponse> SearchLibraryAsync(long sectionId, Type type);
+        Task<SearchLibraryResponse> SearchLibraryAsync(long sectionId, Models.Requests.Type type);
```

Comment:
@LukeHagar The class `Type` causes for ambiguous error as this already exists in C#, so the spec should be updated to a more specific term than type then this will change as well